plugins/colorschemes/modus.nix

```
@@ -0,0 +1,95 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "modus";
+  isColorscheme = true;
+  originalName = "modus-themes-nvim";
+  defaultPackage = pkgs.vimPlugins.modus-themes-nvim;
```

Add `luaName = "modus-themes"` to fix requiring the plugin.

Personally, I'd re-order these as follows, but OTTOMH IDK if that's consistent with what we do in other plugins:


```suggestion
  name = "modus";
  luaName = "modus-themes";
  originalName = "modus-themes-nvim";
  defaultPackage = pkgs.vimPlugins.modus-themes-nvim;
  isColorscheme = true;
```